Pipfile

```diff
@@ -4,26 +4,27 @@ url = "https://pypi.org/simple"
 verify_ssl = true
 
 [dev-packages]
+awscli = "*"
```

Comment:
Why do we need this one (especially if we have boto3 otherwise?)

Reply:
We don't use it in the code nor the tests, but I need it in my venv to call `aws sts get-session-token` for MFA when using Terraform plan/apply :D